dist/fabric.js

```diff
@@ -17572,6 +17645,37 @@ fabric.util.object.extend(fabric.Object.prototype, /** @lends fabric.Object.prot
       return this;
     },
 
+    /**
+     * Draws borders of an object's bounding box.
+     * Requires public properties: width, height
+     * Requires public options: padding, borderColor
+     * @param {CanvasRenderingContext2D} ctx Context to draw on
+     * @param {Object} styleOverride object to override the object style
+     * @return {fabric.Object} thisArg
+     * @chainable
+     */
+    drawIndicationBorders: function(ctx, styleOverride) {
+      if (!styleOverride.indicationBorderColor && !this.indicationBorderColor) {
+        return;
+      }
+      styleOverride = styleOverride || {};
+      var wh = this._calculateCurrentDimensions(),
+          strokeWidth = this.borderScaleFactor,
+          width = wh.x + strokeWidth,
+          height = wh.y + strokeWidth;
+      ctx.save();
+      ctx.strokeStyle = styleOverride.indicationBorderColor || this.indicationBorderColor;
+      ctx.strokeRect(
+        -width / 2,
+        -height / 2,
+        width,
+        height
+      );
+
+      ctx.restore();
+      return this;
+    },
+
```

Comment:
still need to keep this around?

Reply:
Yeah, this is the format for all functions and object variables - Note that it's just the documentation at the top for the description and params that is commented (not sure why github is coloring it ALL as comment....)